pkg/execution/executor/executor.go

```diff
@@ -887,9 +887,9 @@ func (e *executor) Execute(ctx context.Context, id state.Identifier, item queue.
 		))
 	}
 
-	ctx, span := telemetry.NewSpan(ctx,
+	_, span := telemetry.NewSpan(ctx,
```

Comment:
not passing down the execution span context anymore, so all spans will be a flat structure under the function span.